auraed/src/logging/log_channel.rs

```diff
@@ -38,8 +38,8 @@ use tokio::sync::broadcast::{self, Receiver, Sender};
 /// Abstraction Layer for one log generating entity
 /// LogChannel provides channels between Log producers and log consumers
 #[derive(Clone, Debug)]
-pub(crate) struct LogChannel {
```

Comment:
Is this struct being used outside the crate?

Reply:
We wouldn't be able to expose it in `ObserverService::new` if it is crate-private that's why I made the change. Before the `ObserveService` wasn't actually used.

Reply:
We'll refactor in a future PR